.github/workflows/lint.yml

```diff
@@ -18,8 +18,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check formatting with Prettier
+      - name: Reformat with Prettier
```

Comment:
```suggestion
      - name: Check formatting with Prettier
```

We are still using this to check formatting, even if we use a convoluted approach now.

Reply:
This step now reformats the code and then git diff actually checks if it changed the files.